sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.getStudentApplicationSummary.e2e-spec.ts

```diff
@@ -0,0 +1,191 @@
+      collegeCLocation,
+    );
+    applicationRepo = appDataSource.getRepository(Application);
+    studentAssessmentRepo = dataSource.getRepository(StudentAssessment);
+    educationProgramOfferingRepo = dataSource.getRepository(
+      EducationProgramOffering,
+    );
+    institutionLocationRepo = dataSource.getRepository(InstitutionLocation);
+  });
+
+  it("Should get the student application details as summary when student has a submitted application for the institution (application with location id saved).", async () => {
+    // Arrange
+
+    // Student has a submitted application to the institution.
+    const student = await saveFakeStudent(appDataSource);
+    const application = createFakeApplication({
+      location: collegeCLocation,
+      student,
+    });
+    const savedApplication = await applicationRepo.save(application);
+    // Original assessment.
+    const fakeOriginalAssessment = createFakeStudentAssessment({
+      auditUser: student.user,
+    });
+    fakeOriginalAssessment.application = savedApplication;
+    // Offering.
+    const fakeOffering = createFakeEducationProgramOffering({
+      auditUser: student.user,
+    });
+    fakeOffering.offeringIntensity = OfferingIntensity.fullTime;
+    const savedOffering = await educationProgramOfferingRepo.save(fakeOffering);
+    fakeOriginalAssessment.offering = savedOffering;
+    const savedOriginalAssessment = await studentAssessmentRepo.save(
+      fakeOriginalAssessment,
+    );
+    savedApplication.currentAssessment = savedOriginalAssessment;
+    await applicationRepo.save(savedApplication);
+
+    const endpoint = `/institutions/student/${student.id}/application-summary?page=0&pageLimit=10`;
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeCUser,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        results: [
+          {
+            id: application.id,
+            applicationNumber: application.applicationNumber,
+            studyStartPeriod:
+              application.currentAssessment.offering.studyStartDate,
+            studyEndPeriod: application.currentAssessment.offering.studyEndDate,
+            applicationName: "Financial Aid Application",
+            status: application.applicationStatus,
+          },
+        ],
+        count: 1,
+      });
+  });
+
+  it("Should get an empty array as application summary when student has a draft application for the institution (application without the location id saved(i.e draft/cancelled draft status)).", async () => {
+    // Arrange
+
+    // Student has a draft application to the institution.
+    const student = await saveFakeStudent(appDataSource);
+    const application = createFakeApplication({
+      student,
+    });
+    application.data.selectedLocation = collegeCLocation.id;
```

Comment:
`application.data.selectedLocation` has any effect on the test?

Reply:
I think that she's trying to emphasizing that `application.data.selectedLocation` doesn't change the result here.